mount-file.bash

```diff
@@ -10,14 +10,15 @@ shopt -s inherit_errexit  # Inherit the errexit option status in subshells.
 trap 'echo Error when executing ${BASH_COMMAND} at line ${LINENO}! >&2' ERR
 
 # Get inputs from command line arguments
-if [[ $# != 3 ]]; then
+if [[ $# != 4 ]]; then
     echo "Error: 'mount-file.bash' requires *three* args." >&2
```

Comment:
Should read `*four* args`, however may be worth documenting the argument names/description in the error.

Reply:
Ah, yes, of course! Well, it's just a sanity check. The script is not really supposed to be used interactively, so this is just to catch a situation where we're not feeding it all the required arguments programmatically.